packages/react/src/Select/SelectPlain.tsx

```diff
@@ -0,0 +1,55 @@
+import { forwardRef } from 'react';
+import { ChevronDown } from '@obosbbl/grunnmuren-icons';
+import { cx } from '@/utils';
+
+export interface SelectPlainProps
+  extends Omit<React.ComponentPropsWithoutRef<'select'>, 'size'> {
+  /**
+   * Collection of <option />-elements
+   */
+  children: React.ReactNode;
+
+  /** Render select as invalid. Sets `aria-invalid` to true */
+  isInvalid?: boolean;
+
+  /**
+   * Changes font-size, padding and gaps
+   * @default medium
+   */
+  size?: 'medium' | 'small';
+}
+
+export const SelectPlain = forwardRef<HTMLSelectElement, SelectPlainProps>(
+  (props, ref) => {
+    const { children, className, isInvalid, size, ...rest } = props;
+
+    const isSmall = size === 'small';
+
+    return (
+      <div
+        className={cx(className, 'relative', isSmall ? 'text-sm' : undefined)}
+      >
+        <select
+          aria-invalid={isInvalid}
+          {...rest}
+          className={cx(
+            'w-full cursor-pointer appearance-none border border-b-[3px] bg-white focus:-mt-0.5 focus:-ml-0.5 focus:border-[3px] focus:shadow focus:outline-none',
+            isSmall ? 'rounded px-3.5 py-2' : 'rounded-lg px-4 py-3.5',
+            isInvalid
+              ? 'border-red focus:border-red'
+              : 'focus:border-blue-dark border-black',
```

Comment:
Noe grunn til at du går bort fra å bruke objekter? Kunne ikke ene være standard i lista over, mens det som er for isSmall kunne vært en del av objektet?

Reply:
Det funker vel ikke helt her. Det er ikke small som i media queries, så her får klassene lik helt specificity. Da må jeg i så fall bruke important for å få small varianten sine klasser til å gjelde.

Reply:
lawl, stemmer det. Vi har jo to linjer for invalid og !invalid på inputfeltene. Da lener jeg mot denne måten